examples/quickstart.ipynb

```diff
@@ -595,19 +595,22 @@
     {
      "data": {
       "application/vnd.jupyter.widget-view+json": {
-       "model_id": "a3858eeaf66544e9adb8262116a52ec2",
+       "model_id": "2233550e79df49b785e1908e6c23dfc5",
        "version_major": 2,
        "version_minor": 0
       },
       "text/plain": [
-       "Widget(value='<iframe src=\"http://localhost:43247/index.html?ui=P_0x7fbec55233d0_0&reconnect=auto\" class=\"pyvi…"
+       "Widget(value='<iframe src=\"http://localhost:43371/index.html?ui=P_0x7effeb9ce9d0_0&reconnect=auto\" class=\"pyvi…"
       ]
      },
      "metadata": {},
      "output_type": "display_data"
     }
    ],
    "source": [
+    "# NBVAL_SKIP\n",
+    "# tag this cell to be skipped during testing\n",
+    "\n",
```

Comment:
Please remove the comment and instead use a cell tag. (Less confusion for users when reading the notebook)